src/components/atoms/Link/Link.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link as RouterLink } from "react-router-dom";
+import styled, { css } from "styled-components";
+
+const StyledLink = styled(RouterLink)(
+  ({ color, backgroundColor, borderColor }) => css`
+    color: ${color};
+    background-color: ${backgroundColor};
+    border-color: ${borderColor};
+  `,
+);
+
+const Link = ({ children, to = "#", color = "#fff", backgroundColor = "transparent", borderColor = "#0d6efd" }) => {
+  return (
+    <StyledLink to={to} color={color} backgroundColor={backgroundColor} borderColor={borderColor}>
```

Comment:
backgroundColor and borderColor trigger error
<img width="602" alt="image" src="https://user-images.githubusercontent.com/79587920/234690961-950e84d2-0d59-45de-8a24-b4cb78f3a432.png">
you can add $ to the start of prop name or call it in lowercase`backgroundcolor`